components/fxa-client/ios/FxAClient/RustFxAFFI.h

```diff
@@ -80,6 +80,13 @@ FxARustBuffer fxa_profile(FirefoxAccountHandle handle,
                           bool ignore_cache,
                           FxAError *_Nonnull out);
 
+void fxa_migrate_from_session_token(FirefoxAccountHandle handle,
+                                    const char *_Nonnull sessionToken,
+                                    const char *_Nonnull kSync,
+                                    const char *_Nonnull kXCS,
+                                    uint8_t copySessionToken,
+                                    FxAError *_Nonnull out);
+
```

Comment:
We have some code on master that exposes this (and a bunch of additional FxA methods) to Swift, I'm grabbed this one into the point-release so that we don't forget it when merging back to master.